src/etc/inc/filter.inc

```diff
@@ -487,13 +487,13 @@ function filter_configure_sync($verbose = false, $load_aliases = true)
         if (strstr("$rules_error[0]", "busy")) {
             exec('/sbin/pfctl -d; /sbin/pfctl -e; /sbin/pfctl -f /tmp/rules.debug');
             log_error('PF was wedged/busy and has been reset.');
-            file_notice(gettext('PF was wedged/busy and has been reset.'));
+            file_put_contents('/tmp/rules.warning', '/ui/diagnostics/log/core/system');
```

Comment:
Although it's nice to point to a page I think this is somehow losing context about what exactly threw the notice here. Maybe a syslog message prefix wold make sense to filter for this when opening the page?

Reply:
It could also make sense to split off and create a new log facility, requiring less technical details since the current state of the log system doesn't implement passing data to it (to able to filter) from a referring page. Anything else in the firewall section can refer to this as well (such as aliases). Maybe call it "General"?

Reply:
May be better to try and see this in action first. I'm unsure what you mean here.